spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/README.adoc

```diff
@@ -8,25 +8,25 @@ link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-spanner[Sprin
 
 1. Make sure that you have the Cloud SDK configured by following https://cloud.google.com/sdk/docs/[these instructions].
 
-2. Create a new Spanner instanced named "spring-demo" with a database called "trades"
-and a table also named "trades".
-The columns should be created as follows:
-+
+2. Setup the database
+
+a.) Create a new Spanner instance named "spring-demo". For testing, choose the cheapest region possible for you and only 1 region instance should suffice.
+b.) Create with a database called "trades"
```

Comment:
You might want to point out how one creates a database. Either through the UI or via Cloud SDK.